CMakeLists.txt

```diff
@@ -197,7 +206,17 @@ set(EMSCRIPTEN_FASTCOMP ON CACHE BOOL
 set(HERMES_ENABLE_INTL OFF CACHE BOOL
   "Enable JS Intl support (WIP)")
 
-if (HERMES_IS_ANDROID)
+set(HERMES_ENABLE_TEST_SUITE ON CACHE BOOL
+  "Enable the test suite")
+
+set(HERMES_BUILD_APPLE_FRAMEWORK ON CACHE BOOL
+  "Whether to build a framework bundle or dylib on Apple platforms")
```

Comment:
Does this only affect the `libhermes` target? If so, that may be worth mentioning (if not, what does it affect?)

Reply:
Yup, only the libhermes target at this time. Will add 👍